src/build/__main__.py

```diff
@@ -150,6 +157,10 @@ def main(cli_args, prog=None):  # type: (List[str], Optional[str]) -> None
         parser.prog = prog
     args = parser.parse_args(cli_args)
 
+    if args.version:
+        print('build {} ({})'.format(_build.__version__, ', '.join(path for path in _build.__path__ if path)))
```

Comment:
Would it no suffice to just use ``build.__file__`` you can throw an os.path.basename if you must.

Reply:
This makes it more portable and future proof. It can also help debug some weird issues due to namespace colisions.